src/target/riscv/riscv-013.c

```diff
@@ -815,7 +818,7 @@ static int execute_abstract_command(struct target *target, uint32_t command)
 		}
 	}
 
-	if (dmi_write_exec(target, DMI_COMMAND, command) != ERROR_OK)
+	if (dmi_write_exec(target, DMI_COMMAND, command, false) != ERROR_OK)
```

Comment:
So this particular call to dmi_write_exec() doesn't need to make a subsequent scan to ensure success, because the call to wait_for_idle() is about to do another DMI scan which will detect if an error happened?

Reply:
That's right. There are probably other instances of this in the code, but this is one that gets accessed quite a lot.